src/tox_lsr/test_scripts/runcontainer.sh

```diff
@@ -61,10 +63,8 @@ install_requirements() {
     coll_path="$COLLECTION_BASE_PATH/ansible_collections"
     if [ -d "$coll_path/$LOCAL_COLLECTION" ]; then
         info saving local collection at "$coll_path/$LOCAL_COLLECTION"
-        save_tar="$(mktemp)"
+        save_tar="$WORKDIR/save.tar"
         tar cfP "$save_tar" -C "$coll_path" "$LOCAL_COLLECTION"
-        # shellcheck disable=SC2064
-        trap "rm -f $save_tar" RETURN
```

Comment:
`$WORKDIR/save.tar` doesn't need to be cleaned up?

Reply:
or is it that `$WORKDIR` is already being cleaned up?

Reply:
Yes, that's the point -- the top-level `trap` cleans $WORKDIR, so files in it can be created freely. This simplifies the trapping.